src/main/scala-2.12/org/ergoplatform/appkit/ergotool/dex/CreateBuyOrderCmd.scala

```diff
@@ -89,22 +89,40 @@ case class CreateBuyOrderCmd(toolConf: ErgoToolConfig,
 
 
 object CreateBuyOrderCmd extends CmdDescriptor(
-  name = "dex:BuyOrder", cmdParamSyntax = "<wallet file> <buyerAddr> <ergAmount> <tokenId> <tokenAmount>, <dexFee>",
+  name = "dex:BuyOrder", cmdParamSyntax = "<storageFile> <buyerAddr> <ergAmount> <tokenId> <tokenAmount>, <dexFee>",
   description = "put a token buyer order with given <tokenId> and <tokenAmount> to buy at given <ergPrice> price with <dexFee> as a reward for anyone who matches this order with a seller, with <buyerAddr> to be used for withdrawal \n " +
-    "with the given <wallet file> to sign transaction (requests storage password)") {
+    "with the given <storageFile> to sign transaction (requests storage password)") {
+
+  override val parameters: Seq[CmdParameter] = Array(
+    CmdParameter("storageFile", FilePType,
+      "storage with secret key of the sender"),
+    CmdParameter("storagePass", SecretStringPType,
+      "password to access sender secret key in the storage", None,
+      Some(ctx => ctx.console.readPassword("Storage password>"))),
+    CmdParameter("buyerAddr", AddressPType,
+      "address of the buyer"),
```

Comment:
Let's remove this buyerAddr as part of this refactoring. It can be extracted from storage.

Reply:
I planned to do this in my cancel order command branch - #18.